tests/test_wave_01.py

```diff
@@ -137,18 +142,21 @@ def test_moves_movie_from_watchlist_to_watched():
     }
 
     # Act
-    updated_data = watch_movie(janes_data, movie_to_watch["title"])
+    updated_data = watched_movie(janes_data, movie_to_watch)
```

Comment:
Echoing my above comments about modifying tests. This one was modified to use an additional function, but it was supposed to use `watch_movie`. If that wasn't passing, the modification should have been made to that function, rather than adding a new function.